lib/debugcommands/debugcommands.c

```diff
@@ -164,9 +192,34 @@ static int cmd_modify_mem(int argc, const console_cmd_args *argv) {
         size = 1;
     }
 
-    unsigned long address = argv[1].u;
-    unsigned int val = argv[2].u;
+    int argindex = 1;
+    bool read_address = false;
+#if WITH_KERNEL_VM
+    bool phy_addr = false;
+#endif
+
+    while (argc > argindex) {
+#if WITH_KERNEL_VM
+        if (!strcmp(argv[argindex].str, "-p")) {
+            phy_addr = true;
+        } else if (!read_address) {
+#else
```

Comment:
To remove this extra else clause it might be cleaner to do something like
} else
#endif
if (!read_address) {

Reply:
Sure.

Reply:
VISITOR_INFO1_LIVE’